tailwind.config.js

```diff
@@ -41,9 +41,23 @@ module.exports = {
             'light': '#C9C9C9',
         }
       },
+      width: {
+        '100': '25rem', 
+        '104': '26rem',
+        '108': '27rem',
+        '112': '28rem',
+        '116': '29rem',
+        '120': '30rem',
+      },
       height: {
+        '68': '17rem',
+        '84': '21rem',
+        '88': '22rem',
         '100': '25rem', 
+        '104': '26rem',
+        '108': '27rem',
         '112': '28rem',
+        '116': '29rem',
         '120': '30rem',
```

Comment:
There is something weird going on with Tailwind. These height values should be baked in but everytime when I try to use it, it fails to recognize. I had to put these in for it to work...

Did something happen recently that changed some tailwind configs? Maybe our @apply is being overused and it is affecting our tailwind classes?

Reply:
We did just recently enabled purgecss in [another PR](https://github.com/The-Balance-FFXIV/glam/pull/194) — do you think this is impacting that?

Reply:
@nonowazu Ahhh I think that may be it. Reading the purge css documentation there is a way for us to create a safelist. Maybe I'll look into refactoring it later